### ISSUE TITLE
Allow pippin to merge PROBIA designations 

### ISSUE THREAD

**Issue by bap37** [dessn/Pippin#91]
Morning! (or evening)

This is admittedly as much of an SNANA issue as it is a pippin one, but I think the pippin-side fix is easier. 

So for SALT2mu, when using photometric samples, SNANA requires a `varname_pIa` key to work. This varname_pIa key typically corresponds to the output of a classifier. And that's great for a single sample! 

But when combining samples, such as SDSS and PS1, this causes it to immediately crash, since I can either specify the SDSS classifier _or_ the PS1 classifier, but not both.

What I've been doing in the interim is waiting for the 6_BIASCOR step to crash, running a script to rename probabilities in 5_MERGE, then rerunning by hand after copying all the hash.txt values (because otherwise things will get overwritten and it'll crash again!).

While this works, it's a bit ugly, and I think this is gonna end up being a significant issue in the future as we move more towards photometric samples!

**Comment by RickKessler** [comment]
I just learned about this issue; please point to output on disk (or restore if it was clobbered).

**Comment by OmegaLambda1998** [comment]
Proposed solution:

We add a flag to `MERGE` called `MERGE_CLASSIFIERS` which is a list of the classification tasks you want to merge. Then, instead of `PROB_PROBIA_SDSS` and `PROB_PROBIA_PS1` pippin will see that the merged name will be `PROB_PROBIA` as that is the name with the most "overlap".
I think the cleanest way of providing this will be:
```
MERGE:
   MERGE:
      MERGE_CLASSIFIERS:
        - [SNN_PS1, SNN_SDSS]
        - [SCONE_X, SCONE_Y]
```
So that you can group classifiers together.
Then in `BIASCOR` you can either provide a similar list (`CLASSIFIER: [SNN_PS1, SNN_SDSS]`) or pippin can automatically notice that the classifier has been merged (`CLASSIFIER: SNN_PS1`), either way pippin will use the same logic as in `MERGE` to set varname_pIa to `PROB_PROBIA`. This way if you have multiple `BIASCOR` tasks you could specify classifiers which are part of a merge or not as long as the merge output you point BBC to has the right columns

I think I prefer this for a few reasons.
1. It only adds one new flag, and it's a very simple flag
2. You only need to provide Pippin objects without any knowledge of SNANA flags like varname_pIa
3. It leaves a lot of the heavy lifting to Pippin

**Comment by OmegaLambda1998** [comment]
Resolved via f8cfb2d
